src/prop/model.py

```diff
@@ -133,8 +133,12 @@ def PropagationLoss(self, f, lat1, lng1, h1, lat2, lng2, h2, land_cat=''):
         code = self.nlcdIndx.NlcdCode(lat1, lng1)
       land_cat = land_use.NlcdLandCategory(code)
     print 'Using land_cat =', land_cat
-      
-    if land_cat == 'RURAL' or h1 >= 200 or h2 >= 200:
+
```

Comment:
Do you want to add the min-height=20 above? Should it go around line 89? Put a TODO at least...